projects/packages/jitm/src/class-rest-api-endpoints.php

```diff
@@ -1,6 +1,16 @@
 <?php
 /**
- * JITM's REST API Endpoints
+ * JITM's (LEGACY) REST API Endpoint
+ *
+ * DO NOT REMOVE.
+ *
+ * Whilst this endpoint has been superseded by the wpcom/v3/jitm endpoint.
```

Comment:
```suggestion
 * Whilst this endpoint has been superseded by the wpcom/v3/jitm endpoint,
```

Very tiny nitpick, just because the sentence on the next line begins in lowercase, so I'm guessing this was intended to be one sentence.